DESCRIPTION

```diff
@@ -15,6 +15,8 @@ Imports:
   ggplot2,
   irlba,
   tibble,
+  mclust,
+  reshape2,
   umap
```

Comment:
I am getting the following error when I try to install the package in the R Project environment (using `Install`).
Any idea why these specific packages don't work?
<img width="636" alt="Screenshot 2023-09-08 at 11 28 15 PM" src="https://github.com/aashen12/VoxelProc/assets/66567109/d6d5c507-c8b2-4faa-a66c-4c3766cc22c2">

Reply:
I don't know why it's giving that error, because irlba and umap are both in the namespace. I'll look into it more.

Reply:
I can install it from an external environment but not in the R package environment itself

Reply:
I am not getting the error on my computer. Are you sure you've git pulled the most recent version with an updated description and namespace? Additional, you could try running devtools::install("pathname of VoxelProc repository") and update all the dependencies, but I don't know if that will solve the problem.

Reply:
![image](https://github.com/aashen12/VoxelProc/assets/108899680/a6531f23-eb3b-427b-a3b7-0e568c3b4339)

This is what it looks like for me.

Reply:
It works for me now! Thanks!

Reply:
Now I'm getting this error which seems to be different from the last one. It appears to be reading in some data (which may be on my machine). Not sure why this is happening but we can discuss today.

![Screenshot 2023-09-11 at 10 55 20 AM](https://github.com/aashen12/VoxelProc/assets/66567109/201d18e3-4aa5-4e63-acec-a6342448b363)